tools/test/examples/elf_float_checker.py

```diff
@@ -108,6 +123,7 @@ def check_action(args):
         print("Found float symbols:")
         for float_symbol in float_symbols:
             print(float_symbol)
+        raise FloatSymbolsFound("Found float symbols in {}".format(args.elf_file))
     else:
         print("No float symbols found.")
```

Comment:
shall we change the message to `Passed - No float symbols found` make the output to be explicit . the original message more looks like warning or error. 

Similarly, above messages better to be `Failed - Found float symbols: `

Reply:
OK

Reply:
[Done](https://github.com/ARMmbed/mbed-os/pull/11429#issuecomment-540944958).